### DISCUSSION TITLE
Hello, may I ask, what is the reason behind the error "lxml.etree.ParserError: Document is empty"?

### DISCUSSION THREAD

**Discussion by sanceca** [sometastycake/steam-password-change#3] [Q&A]
File "E:\Work\steampw\venu\lib\site-packages\lxml\html\__init__.py", line 762, in document_fromstring
    "Document is empty")
lxml.etree.ParserError: Document is empty
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x0000013462FA2BA8>

**Comment by mashtalir** [comment]
@sanceca did you fix that?
